docs/getting_started/bridge_console.md

```diff
@@ -3,59 +3,155 @@ title: Bridge
 sidebar_position: 4
 ---
 
-The NEAR to Calimero bridge enables users move assets between NEAR public networks (Testnet or Mainnet) and your Private Shard. Additionally, you can execute cross shard contract calls which includes private smart contracts calls inside the Shard from a public contract or public smart contract calls from a contract inside the Private Shard.
+The Calimero bridge serves as a vital tool for users to transfer assets, including fungible tokens (FTs) and non-fungible tokens (NFTs), between the NEAR public network (Testnet or Mainnet) and their private shard. Additionally, it facilitates cross-shard contract calls, allowing users to invoke smart contracts from a public contract into the shard.
```

Comment:
I know we agreed that we will use the "Calimero chain" notation instead of the "Calimero shard" notation for internal usage. Are we still using "Calimero shard" and "cross-shard contract call"?

I am ok with it being both ways